package.json

```diff
@@ -33,25 +33,28 @@
     "@semantic-release/npm": "7.1.3",
     "@semantic-release/release-notes-generator": "9.0.3",
     "@types/fs-extra": "9.0.11",
-    "@types/jest": "26.0.23",
-    "@types/jest-image-snapshot": "4.3.0",
-    "@types/puppeteer": "5.4.3",
+    "@types/jest": "29.5.11",
+    "@types/jest-image-snapshot": "6.4.0",
```

Comment:
I suggest keeping these lists alphabetically sorted.

Reply:
I think it is still unsorted. If you would like, you can use the VS Code `Sort lines ascending` command after selecting the lines in question.